.azure/gpu-tests-lite.yml

```diff
@@ -0,0 +1,126 @@
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - "master"
+      - "release/*"
+      - "refs/tags/*"
+
+pr:
+  - "master"
+  - "release/*"
+
+variables:
+  - name: continue
+    value: '1'
+
+jobs:
+  - job: testing
+    strategy:
+      matrix:
+        'PyTorch - stable':
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.12-cuda11.6.1"
+    # how long to run the job before automatically cancelling
+    timeoutInMinutes: "80"
+    # how much time to give 'run always even if cancelled tasks' before stopping them
+    cancelTimeoutInMinutes: "2"
+    pool: azure-jirka-spot
+    container:
+      image: $(image)
+      # default shm size is 64m. Increase it to avoid:
+      # 'Error while creating shared memory: unhandled system error, NCCL version 2.7.8'
+      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all --shm-size=512m"
+    workspace:
+      clean: all
+
+    steps:
+
+    - bash: |
+        CHANGED_FILES=$(git diff --name-status origin/master -- . | awk  '{print $2}')
+        FILTER='.azure/gpu_*|src/lightning_lite|requirements/lite|tests/tests_lite'
+        echo $CHANGED_FILES > changed_files.txt
+        MATCHES=$(cat changed_files.txt | grep -E $FILTER)
+        echo $MATCHES
+        if [ -z "$MATCHES" ]; then
+            echo "Skip"
+            echo "##vso[task.setvariable variable=continue]0"
+        else
+            echo "Continue"
+            echo "##vso[task.setvariable variable=continue]1"
+        fi
+      displayName: Skipper
+
+    - bash: |
+        lspci | egrep 'VGA|3D'
+        whereis nvidia
+        nvidia-smi
+        which python && which pip
+        python --version
+        pip --version
+        pip list
+      displayName: 'Image info & NVIDIA'
+      condition: eq(variables['continue'], '1')
+
+    - bash: |
+        set -e
+        TORCH_VERSION=$(python -c "import torch; print(torch.__version__.split('+')[0])")
+        CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
+        python ./requirements/pytorch/adjust-versions.py requirements/pytorch/lite.txt ${PYTORCH_VERSION}
+        pip install -e .[strategies] --find-links https://download.pytorch.org/whl/cu${CUDA_VERSION_MM}/torch_stable.html
+        pip install --requirement requirements/pytorch/devel.txt --find-links https://download.pytorch.org/whl/cu${CUDA_VERSION_MM}/torch_stable.html
+        pip list
+      env:
+        PACKAGE_NAME: pytorch
+        FREEZE_REQUIREMENTS: 1
+      displayName: 'Install dependencies'
+      condition: eq(variables['continue'], '1')
+
+    - bash: |
+        set -e
+        python requirements/collect_env_details.py
+        python -c "import torch ; mgpu = torch.cuda.device_count() ; assert mgpu >= 2, f'GPU: {mgpu}'"
+        python requirements/pytorch/check-avail-strategies.py
```

Comment:
Lite does not support Bagua no? It shouldn't install it, which will make this check fail

Reply:
Yes, I removed all bagua install references. Are you saying I should delete
python requirements/pytorch/check-avail-strategies.py ?

Reply:
You can delete it or create an equivalent one that is for Lite deps